### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by yzl19940819**
add EISeg English doc

### CODE REVIEW

File: EISeg/README.md [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
+[![Python 3.6](https://img.shields.io/badge/python-3.6+-blue.svg)](https://www.python.org/downloads/release/python-360/)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)](LICENSE)
 [![Downloads](https://pepy.tech/badge/eiseg)](https://pepy.tech/project/eiseg)
 <!-- [![GitHub release](https://img.shields.io/github/release/Naereen/StrapDown.js.svg)](https://github.com/PaddleCV-SIG/iseg/releases) -->
 
+English | [简体中文](README_CN.md)
+## Latest Developments
 
-## 最新动向
+- Our paper on interactive segmentation named [EdgeFlow](https://arxiv.org/abs/2109.09406) is accepted by ICCV 2021 Workshop.
```
Comment (haoyuying) [inline EISeg/README.md:11]: has been accepted

File: EISeg/README.md [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
+- We release EISeg 0.3.0 with more functions and support for Polygon editing.
 
-- 交互式分割论文[EdgeFlow](https://arxiv.org/abs/2109.09406)被ICCV 2021 Workshop接收。
-- 支持多边形编辑，上线更多功能，最新EISeg 0.3.0推出。
+## Introduction
 
-## 介绍
-
-EISeg(Efficient Interactive Segmentation)是基于飞桨开发的一个高效智能的交互式分割标注软件。涵盖了高精度和轻量级等不同方向的高质量交互式分割模型，方便开发者快速实现语义及实例标签的标注，降低标注成本。 另外，将EISeg获取到的标注应用到PaddleSeg提供的其他分割模型进行训练，便可得到定制化场景的高精度模型，打通分割任务从数据标注到模型训练及预测的全流程。
+EISeg (Efficient Interactive Segmentation) is an efficient and intelligent interactive segmentation annotation software developed based on PaddlePaddle. It covers a large number of high-quality segmentation models in different directions such as *high-performance* and *lightweight*, providing convenience to the rapid annotation of  semantic and instance labels with reduced cost. In addition, by applying the annotations obtained by EISeg to other segmentation models provided by PaddleSeg for training, high-performance models with customized scenarios can be created, integrating the whole process of segmentation tasks from data annotation to model training and prediction.
```
Comment (haoyuying) [inline EISeg/README.md:16]: intelligent interactive segmentation tool
It covers several high-quality segmentation models in different directions

File: EISeg/README.md [hunk 3]
```diff
@@ -1,61 +1,61 @@
...
 
 ![eiseg_demo](../docs/images/eiseg_demo.gif)
 
-## 模型准备
+## Model Preparation
 
-在使用EIseg前，请先下载模型参数。EISeg开放了在COCO+LVIS和大规模人像数据上训练的四个标注模型，满足通用场景和人像场景的标注需求。其中模型结构对应EISeg交互工具中的网络选择模块，用户需要根据自己的场景需求选择不同的网络结构和加载参数。
+Please download the model parameters before using EIseg. EISeg provides four annotation models trained on COCO+LVIS and large-scale portrait data to meet the needs of both generic and portrait scenarios. The model architecture corresponds to the network selection module in EISeg interactive tools, and users need to select different network structures and loading parameters in accordance with their own needs.
```
Comment (haoyuying) [inline EISeg/README.md:22]: elect different network structures and load parameters

File: EISeg/README.md [hunk 4]
```diff
@@ -1,61 +1,61 @@
...
 
 ```shell
 git clone https://github.com/PaddlePaddle/PaddleSeg.git
 ```
 
-安装好所需环境后，进入EISeg，可通过直接运行eiseg打开EISeg：
+Enable EISeg by running eiseg after installing the needed environment:
 
 ```shell
 cd PaddleSeg\contrib\EISeg
```
Comment (haoyuying) [inline EISeg/README.md:54]: cd PaddleSeg\EISeg

File: EISeg/README.md [hunk 5]
```diff
@@ -64,103 +64,101 @@ python exe.py
...
+
 ```shell
 eiseg
 ```
-即可运行软件。
 
-### Windows exe
-
-EISeg使用[QPT](https://github.com/GT-ZhangAcer/QPT)进行打包。可以从[这里](https://paddleseg.bj.bcebos.com/eiseg/EISeg0.3.0.1.7z)下载最新EISeg。解压后双击启动程序.exe即可运行程序。程序第一次运行会初始化安装所需要的包，请稍等片刻。
+Now, you can run pip.
```
Comment (haoyuying) [inline EISeg/README.md:79]: run eiseg.

File: EISeg/README.md [hunk 6]
```diff
@@ -64,103 +64,101 @@ python exe.py
...
-3. 如果有图像之间名称相同但后缀名不同，可以打开`标签和图像使用相同扩展名`。
-4. 还可设置灰度保存、伪彩色保存和抠图保存，见工具栏中7-9号工具。
+1. Click Space key to complete interactive annotation, then appears the polygon boundary; when you need to continue the interactive process inside the polygon, click Space to switch to interactive mode so the polygon cannot be selected and changed.
+2. The polygon can be dragged and deleted. Use the left mouse to drag the anchor point, double-click the anchor point to delete it, and double-click a side to add an anchor point.
+3. With `Keep Maximum Connected Blocks` on, only the largest area will remain in the image, the rest of the small areas will not be displayed and saved.
 
-- **生成mask**
+- **Save Format**
 
-1. 标签按住第二列可以进行拖动，最后生成mask时会根据标签列表从上往下进行覆盖。
```
Comment (haoyuying) [inline EISeg/README.md:140]: 删掉

File: EISeg/README.md [hunk 7]
```diff
@@ -64,103 +64,101 @@ python exe.py
...
+4. You can also save as grayscale,  pseudo-color or matting image, see tools 7-9 in the toolbar
 
-- **界面模块**
+- **Generate mask**
 
-1. 可在`显示`中选择需要显示的界面模块，正常退出时将会记录界面模块的状态和位置，下次打开自动加载。
+1. Labels can be dragged by holding down the second column, and the final generated mask will be overwritten from top to bottom according to the label list.
 
-## 工具
+- **Interface Module**
```
Comment (haoyuying) [inline EISeg/README.md:145]: 删掉

File: EISeg/README.md [hunk 8]
```diff
@@ -1,61 +1,61 @@
...
 ```shell
 cd PaddleSeg\contrib\EISeg
 python -m eiseg
 ```
 
-或进入eiseg，运行exe.py打开EISeg：
+Or you can run exe.py in eiseg:
 
 ```shell
 cd PaddleSeg\contrib\EISeg\eiseg
```
Comment (haoyuying) [inline EISeg/README.md:61]: cd PaddleSeg\EISeg\eiseg

File: EISeg/README.md [hunk 9]
```diff
@@ -64,103 +64,101 @@ python exe.py
...
-- **界面模块**
+- **Generate mask**
 
-1. 可在`显示`中选择需要显示的界面模块，正常退出时将会记录界面模块的状态和位置，下次打开自动加载。
+1. Labels can be dragged by holding down the second column, and the final generated mask will be overwritten from top to bottom according to the label list.
 
-## 工具
+- **Interface Module**
 
-有关工具使用参考[这里](tool/README.md)，目前包括：
```
Comment (haoyuying) [inline EISeg/README.md:148]: 这部分没翻译

File: EISeg/README.md (hunk 2 above)
Comment (linhandev) [reply EISeg/README.md:16]: in different directions 感觉 for different use cases 更好

Comment (linhandev) [reply EISeg/README.md:16]: It covers a large number of high-quality segmentation models in different directions such as high-performance and lightweight, providing convenience to the rapid annotation of semantic and instance labels with reduced cost. 

It includes a large number of high-quality segmentation models for different use cases, including high-resolution and lightweight models, generating accurate semantic and instance segmentation annotation rapidly with reduced cost.

File: EISeg/README.md (hunk 3 above)
Comment (linhandev) [reply EISeg/README.md:22]: The model architecture corresponds to the network selection module in EISeg interactive tools, and users need to select different network structures and loading parameters in accordance with their own needs.

You can select different architectures, high-resolution or lightweight and corresponding parameters, for generic and portrait based on the nature of the annotated data.

**Pull request closed without merging**